recipe/meta.yaml

```diff
@@ -23,10 +23,10 @@ source:
     - pycrypto_PR_200.diff
 
 build:
-  number: 0
-  skip: true  # [py>35]
+  number: 1
   script:
-    - export LDFLAGS="-headerpad_max_install_names"  # [osx]
+    - set CL="/FI %VCINSTALLDIR%\\include\\stdint.h %CL%"  # [win and py>=35]
```

Comment:
This [forces]( https://msdn.microsoft.com/en-us/library/8c5ztk84.aspx ) a particular copy of `stdint.h` to be used by all `#include` directives in the C code. Should note this happens in spite of the `stdint.h` that PyCrypto [includes]( https://github.com/dlitz/pycrypto/blob/v2.6.1/src/inc-msvc/stdint.h ).